recipes/cdna_cupcake/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set version = "5.3" %}
+{% set name = "cDNA_Cupcake" %}
+{% set sha256 = "a5bde38f829131e705395b6a6d2f43f2bb208f6e27a111b57ecd6e80443d26a8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: tofu2_v21.tar.gz
+  url: https://github.com/Magdoll/cDNA_Cupcake/archive/tofu2_v21.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - py.patch
+
+build:
+  number: 0
+  skip: True  # [py3k]
+
+requirements:
+  build:
+    - python
+    - biopython
+    - bx-python <=0.7.3
+
+  run:
+    - python
+    - biopython
+    - bx-python <=0.7.3
+    - r
```

Comment:
r is probably not needed here

Reply:
Thanks for the feedback @bgruening ! There are some R scripts in the package, that's why R is there.

Reply:
You already included r-base, which should be enough in most cases so `r` can be remove.